cmd/agent/app/settings/runtime_setting_dogstatsd_stats.go

```diff
@@ -0,0 +1,55 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package settings
+
+import (
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/common"
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+// dsdStatsRuntimeSetting wraps operations to change log level at runtime.
+type dsdStatsRuntimeSetting string
+
+func (s dsdStatsRuntimeSetting) Description() string {
+	return "Enable/disable the dogstatsd debug stats. Possible values: true, false"
+}
+
+func (s dsdStatsRuntimeSetting) Name() string {
+	return string(s)
+}
+
+func (s dsdStatsRuntimeSetting) Get() (interface{}, error) {
+	return common.DSD.DebugMetricsStats, nil
+}
+
+func (s dsdStatsRuntimeSetting) Set(v interface{}) error {
```

Comment:
Out of the scope of this PR, but I am wondering if using a callback instead of setting the field directly (`common.DSD.DebugMetricsStats = newValue`) was considered when designing dynamic settings?
A dynamic setting can have a `OnValueChanged` callback that is the only way to get the setting. The pro of this, is that it make it clear to the consumer the value can change. What do you think?

Reply:
This kind of solution has been left out of the RuntimeSettings RFC because there was no existing use case so far. There is not any raising for now, but that may be added later if the needs appear.